src/main/java/me/xmrvizzy/skyblocker/skyblock/item/ItemCooldowns.java

```diff
@@ -0,0 +1,116 @@
+package me.xmrvizzy.skyblocker.skyblock.item;
+
+import com.google.common.collect.ImmutableList;
+import me.xmrvizzy.skyblocker.config.SkyblockerConfigManager;
+import me.xmrvizzy.skyblocker.events.ClientPlayerBlockBreakEvent;
+import me.xmrvizzy.skyblocker.utils.ItemUtils;
+import net.fabricmc.fabric.api.event.player.UseItemCallback;
+import net.minecraft.block.BlockState;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.registry.tag.BlockTags;
+import net.minecraft.sound.BlockSoundGroup;
```

Comment:
Don't mean to be picky but can you just accept this.
```suggestion
```

Reply:
removed 👍